DataFormats/Detectors/ITSMFT/common/src/TopologyDictionary.cxx

```diff
@@ -155,6 +162,15 @@ ClassImp(o2::itsmft::TopologyDictionary)
       return mVectorOfGroupIDs[n].mPattern;
   }
 
+  bool TopologyDictionary::IsGroup(int n)
+  {
+    if (n < 0 || n >= (int)mVectorOfGroupIDs.size()) {
```

Comment:
@shahor02 I have still to implement this correction, because the method is called by the class LookUp. If I declare it inline, I have some linking problem when compiling LookUp.
The check for n<=0 has been removed

Reply:
Done.